.github/workflows/test.yml

```diff
@@ -872,6 +872,40 @@ jobs:
       matrix:
         spec:
           [
+            './portal/cypress/e2e/journeys/*.spec.js',
+            './portal/cypress/e2e/journeys/homepage/**/*.spec.js',
+            './portal/cypress/e2e/journeys/admin/**/*.spec.js',
+            './portal/cypress/e2e/journeys/checker/**/*.spec.js',
+            './portal/cypress/e2e/journeys/dashboard/dashboard-deals/filter-by/*.spec.js',
+            './portal/cypress/e2e/journeys/dashboard/dashboard-deals/filters/*.spec.js',
+            './portal/cypress/e2e/journeys/dashboard/dashboard-deals/sort/*.spec.js',
+            './portal/cypress/e2e/journeys/dashboard/dashboard-facilities/filter-by/*.spec.js',
+            './portal/cypress/e2e/journeys/dashboard/dashboard-facilities/filters/*.spec.js',
+            './portal/cypress/e2e/journeys/dashboard/dashboard-facilities/sort/*.spec.js',
+            './portal/cypress/e2e/journeys/feedback/**/*.spec.js',
+            './portal/cypress/e2e/journeys/login/**/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/check-deal-details/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/clone-deal/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/dashboard/**/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/submit-a-deal-for-review/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/submit-a-deal-for-review-with-issued-and-unissued-facilities-and-resubmit-after-checker-returns/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/submit-issued-facilities-for-review/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/bank-details/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/eligibility/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/abandon-a-deal/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/before-you-start/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/bss-mandatory-criteria/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/edit-deal-name/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/proceed-to-review/*.spec.js',
+            './portal/cypress/e2e/journeys/maker/select-scheme/*.spec.js',
```

Comment:
Out of interest, is there a reason we don't do e.g.:

`'./portal/cypress/e2e/journeys/maker/**/*.spec.js',`

instead of e.g. 

```
'./portal/cypress/e2e/journeys/maker/eligibility/*.spec.js',
'./portal/cypress/e2e/journeys/maker/abandon-a-deal/*.spec.js',
'./portal/cypress/e2e/journeys/maker/before-you-start/*.spec.js',
...
```

(I might not understand the **/* correctly though)

Reply:
I believe these are some of the tests @ttbarnes split up to reduce e2e run time - its starting to look a bit ugly and maybe there is a better way of dealing with this 🤷

Reply:
Correct - they are split up to reduce E2E time. Prior to this change, some individual E2E runners took I think 20-30mins to run 😱 Last time I checked, I think the entire suite took ~10-12mins, quicker than the API tests.

The trade off as you've alluded to, is extra maintenance with the `test.yml`, but I think it's worth it - just need to remember to update it if there is a new E2E directory, or if a directory becomes busy.

I am not sure how the `test.yml` could become prettier, it's all about the directories and wildcards for those directories could be uglier. Open to any ideas as always though 😊